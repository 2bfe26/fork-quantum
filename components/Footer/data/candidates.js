export default {
  id: 1,
  title: 'Candidatos',
  cols: 8,
  theme: {
    primaryColor: '#005da4',
    secondaryColor: '#666666',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '12px',
  },
  linkGroups: [{
    id: 1,
    sideSeparator: true,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Buscar vagas',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Por palavra-chave',
      href: '/vagas/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Por área/cargo',
      href: 'https://www.catho.com.br/vagas/por-area/',
      target: '_self',
      img: null,
    }, {
      id: 4,
      title: false,
      text: 'Melhores e maiores empresas do Brasil',
      href: 'https://www3.catho.com.br/vag/empregos-maiores-empresas/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 2,
    sideSeparator: true,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Produtos Gratuitos',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Aviso de vagas e buscas salvas',
      href: '/aviso-vagas/listagem/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Buscar vagas',
      href: 'https://www.catho.com.br/vagas/',
      target: '_self',
      img: null,
    }, {
      id: 4,
      title: false,
      text: 'Guia de Profissões e Salários',
      href: '/profissoes/',
      target: '_self',
      img: null,
    }, {
      id: 5,
      title: false,
      text: 'Por dentro das empresas',
      href: '/por-dentro/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 3,
    sideSeparator: false,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Apoio ao candidato',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Elaboração de Currículo',
      href: '/https://parceiros.catho.com.br/gop.php?origem=cat&a=6515&p=12&url1=1&url=servicoscatho%2Fcarreira%2Felaboracao-de-curriculo%2F',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Análise de Currículo',
      href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6523&p=4&url1=1&url=servicoscatho%2Fcarreira%2Fanalise-de-curriculo%2F',
      target: '_self',
      img: null,
    }, {
      id: 4,
      title: false,
      text: 'Simulação de Entrevista',
      href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6527&p=16&url1=1&url=servicoscatho%2Fcarreira%2Fsimulacao-de-entrevista%2F',
      target: '_self',
      img: null,
    }, {
      id: 5,
      title: false,
      text: 'Elaboração de Carta de Apresentação',
      href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6519&p=13&url1=1&url=servicoscatho%2Fcarreira%2Fcarta-de-apresentacao%2F',
      target: '_self',
      img: null,
    }, {
      id: 6,
      title: false,
      text: 'Conheça Todas as Soluções',
      href: '/servicoscandidatos/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 4,
    sideSeparator: true,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Carreira e sucesso',
      href: 'https://www.catho.com.br/carreira-sucesso/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 5,
    sideSeparator: true,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Exclusivo para assinantes',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Cursos Rápidos',
      href: '/cursos-rapidos/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Cadastro Catho',
      href: 'https://cadastro.catho.com.br/',
      target: '_self',
      img: null,
    }, {
      id: 4,
      title: false,
      text: 'Consultor Virtual',
      href: '/consvirtual/',
      target: '_self',
      img: null,
    }],
  }, {
    id: 6,
    sideSeparator: false,
    cols: 4,
    links: [{
      id: 1,
      title: true,
      text: 'Ajuda',
      href: null,
      target: null,
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Central de Relacionamento',
      href: 'https://www.catho.com.br/sac/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: true,
      text: 'Promoção 7 dias grátis',
      href: null,
      target: null,
      img: null,
    }, {
      id: 5,
      title: false,
      text: 'Entenda a promoção',
      href: 'https://www.catho.com.br/help/usr/faq_promocao.phtml?home=1&width=650px&height=600px&iframe=true',
      target: '_self',
      img: null,
    }],
  }],
};
