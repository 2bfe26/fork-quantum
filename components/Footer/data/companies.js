export default {
  id: 1,
  title: 'Empresas',
  cols: 2,
  theme: {
    primaryColor: '#005C45',
    secondaryColor: '#666666',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '12px',
  },
  linkGroups: [{
    id: 1,
    sideSeparator: false,
    cols: 12,
    links: [{
      id: 1,
      title: false,
      text: 'Solução Atração dos Melhores',
      href: 'https://seguro.catho.com.br/inc/novaInclusao/FormPrincipal.php?tipoProd=Vag&c=13006',
      target: '_self',
      img: null,
    }, {
      id: 2,
      title: false,
      text: 'Solução Recrutamento Perfeito',
      href: 'https://www.catho.com.br/empresas/produtos/',
      target: '_self',
      img: null,
    }, {
      id: 3,
      title: false,
      text: 'Produtos para Empresa',
      href: 'https://www.catho.com.br/empresas/produtos/',
      target: '_self',
      img: null,
    }],
  }],
};
