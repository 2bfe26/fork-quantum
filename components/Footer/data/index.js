import Candidates from './candidates';
import Companies from './companies';
import Courses from './courses';
import Institutional from './institutional';
import Stamps from './stamps';

const Data = {
  theme: {
    primaryColor: '#005da4',
    secondaryColor: '#666666',
    borderColor: '#e3e3e3',
    backgroundColor: '#f5f5f5',
    fontFamily: 'Arial',
    fontSize: '12px',
    paddingTop: '15px',
    paddingBottom: '10px',
    StampsHolderPadding: '10px',
  },
  candidates: Candidates,
  companies: Companies,
  courses: Courses,
  institutional: Institutional,
  stamps: Stamps,
};

export default Data;
