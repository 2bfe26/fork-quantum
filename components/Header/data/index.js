import Segments from './segments';
import Promotion from './promotion';
import Menu from './menu';

const Data = {
  theme: {
    primaryColor: '#005da4',
    secondaryColor: '#666666',
    borderColor: '#e3e3e3',
    backgroundColor: '#f5f5f5',
    fontFamily: 'Arial',
    fontSize: '12px',
    paddingTop: '10px',
    paddingBottom: '10px',
    StampsHolderPadding: '10px',
  },
  segments: Segments,
  promotion: Promotion,
  menu: Menu,
};

export default Data;
