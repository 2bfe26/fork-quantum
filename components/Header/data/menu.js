const common = [
  {
    id: 3,
    title: false,
    text: 'Aviso de vagas',
    href: '/aviso-vagas/listagem/',
    target: '_self',
    img: null,
  }, {
    id: 4,
    title: false,
    text: 'Envie seu Currículo por 7 dias grátis',
    href: 'https://seguro.catho.com.br/inclusao/curriculo.php?nova_inclusao=1&usr[plano_id]=1&cur[perfil_id]=1&cur[use_personalizado]=0&a=0&ppd=0&tipoinclusao=807',
    target: '_self',
    img: null,
  }, {
    id: 5,
    title: false,
    text: 'Guia de profissões',
    href: '/profissoes/',
    target: '_self',
    img: null,
  }, {
    id: 6,
    title: null,
    text: 'Por dentro das empresas',
    href: '/por-dentro/',
    target: '_self',
    img: null,
  }, {
    id: 7,
    title: null,
    text: 'Portal do currículo',
    href: 'https://www.catho.com.br/portaldocurriculo/',
    target: '_self',
    img: null,
  }, {
    id: 8,
    title: null,
    text: 'Vagas de estágio',
    href: 'https://www.catho.com.br/estagio/',
    target: '_self',
    img: null,
  }, {
    id: 9,
    title: 'Exclusivo para assinantes',
    text: null,
    href: null,
    target: null,
    img: null,
  }, {
    id: 10,
    title: null,
    text: 'Cursos rápidos',
    href: '/cursos-rapidos/',
    target: '_self',
    img: null,
  }, {
    id: 11,
    title: null,
    text: 'Cadastro Catho',
    href: 'https://cadastro.catho.com.br/',
    target: '_self',
    img: null,
  }, {
    id: 12,
    title: null,
    text: 'Consultor virtual',
    href: '/consvirtual/',
    target: '_self',
    img: null,
  }, {
    id: 13,
    title: null,
    text: 'Vantagens de assinar a Catho',
    href: '/cadastrar-curriculo/',
    target: '_self',
    img: null,
  }, {
    id: 14,
    title: 'Apoio ao candidato',
    text: null,
    href: null,
    target: null,
    img: null,
  }, {
    id: 15,
    title: null,
    text: 'Elaboração de currículo',
    href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6503&p=12&url1=1&url=servicoscatho/carreira/elaboracao-de-curriculo/',
    target: '_self',
    img: null,
  }, {
    id: 16,
    title: null,
    text: 'Análise de currículo',
    href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6507&p=4&url1=1&url=servicoscatho/carreira/analise-de-curriculo/',
    target: '_self',
    img: null,
  }, {
    id: 17,
    title: null,
    text: 'Simulação de entrevista',
    href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6509&p=16&url1=1&url=servicoscatho/carreira/simulacao-de-entrevista/',
    target: '_self',
    img: null,
  }, {
    id: 18,
    title: null,
    text: 'Elaboração de carta de apresentação',
    href: 'https://parceiros.catho.com.br/gop.php?origem=cat&a=6505&p=13&url1=1&url=servicoscatho/carreira/carta-de-apresentacao/',
    target: '_self',
    img: null,
  }, {
    id: 19,
    title: null,
    text: 'Cursos',
    href: '/curso/',
    target: '_self',
    img: null,
  }, {
    id: 20,
    title: null,
    text: 'Todas as soluções',
    href: '/servicoscandidatos/',
    target: '_self',
    img: null,
  },
];

const desktop = {
  theme: {
    primaryColor: '#666666',
    secondaryColor: '#CCCCCC',
    tertiaryColor: '#333333',
    backgroundColor: '#FFFFFF',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '14px',
    marginLogo: '0px 20px 0px 0px',
    menuHover: {
      borderColor: '#005da4',
      borderSize: '2px',
    },
  },
  links: [{
    id: 1,
    title: false,
    text: 'Buscar vagas',
    href: '/vagas/',
    target: '_self',
    img: null,
  }, {
    id: 2,
    title: false,
    text: 'Guia de profissões',
    href: '/profissoes/',
    target: '_self',
    img: null,
  }, {
    id: 3,
    title: null,
    text: 'Por dentro das empresas',
    href: '/por-dentro/',
    target: '_self',
    img: null,
  }, {
    id: 4,
    title: false,
    text: 'Soluções para candidatos',
    href: null,
    target: null,
    img: null,
    collapsedLinkGroup: common,
  }],
};

const phone = {
  theme: {
    primaryColor: '#666666',
    secondaryColor: '#CCCCCC',
    tertiaryColor: '#333333',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '12px',
  },
  links: [{
    id: 1,
    title: false,
    text: 'Buscar vagas',
    href: '/vagas/',
    target: '_self',
    img: null,
    icon: 'search',
  }, {
    id: 2,
    title: null,
    text: 'Por dentro das empresas',
    href: '/por-dentro/',
    target: '_self',
    img: null,
    icon: 'enterprise',
  }, {
    id: 3,
    title: false,
    text: 'Aviso de vagas',
    href: '/aviso-vagas/listagem/',
    target: '_self',
    img: null,
    icon: 'alert',
  },
  ...common.slice(1)],
};

export default {
  desktop,
  phone,
};
