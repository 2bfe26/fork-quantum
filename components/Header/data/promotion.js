export default {
  theme: {
    primaryColor: '#da1d5d',
    secondaryColor: '#b6154e',
    textColor: '#fff',
  },
};
