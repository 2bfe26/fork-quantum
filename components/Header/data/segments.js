export default {
  id: 1,
  title: null,
  cols: 12,
  theme: {
    primaryColor: '#555',
    secondaryColor: '#CCCCCC',
    tertiaryColor: '#333333',
    titleFontSize: '14px',
    titleFontWeight: 'bold',
    linkFontSize: '12px',
  },
  links: [{
    id: 1,
    title: false,
    text: 'candidatos',
    href: '/',
    target: '_self',
    img: null,
  }, {
    id: 2,
    title: false,
    text: 'empresas',
    href: 'https://www.catho.com.br/empresa/',
    target: '_self',
    img: null,
  }, {
    id: 3,
    title: false,
    text: 'cursos',
    href: 'https://www.catho.com.br/educacao/',
    target: '_self',
    img: null,
  }, {
    id: 4,
    title: false,
    text: null,
    href: 'https://www.catho.com.br/help/',
    target: '_self',
    img: {
      src: 'header-info.svg',
      width: '16px',
      height: '16px',
      alt: '',
    },
  }],
};
