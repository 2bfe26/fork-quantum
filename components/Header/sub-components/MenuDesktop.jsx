import React from 'react';
import PropTypes from 'prop-types';
import styled, { ThemeProvider } from 'styled-components';
import { Container, Row, Col, Hide, BREAKPOINTS } from '../../Grid';
import MenuButton from './MenuButton';
import LoginBox from './LoginBox';

const MenuDesktop = ({
  data, handleMenuOpen, handleLoginOpen, menuOpened, loginOpened,
}) => (
  <ThemeProvider theme={data.menu.desktop.theme}>
    <MenuDesktopContainer fluid className={menuOpened}>
      <Container>
        <Row>
          <StyledCol phone={12} tablet={12} desktop={12} className={menuOpened}>
            <MenuNavigation>
              <Hide desktop large hd>
                <MenuButton handleMenuOpen={handleMenuOpen} menuOpened={menuOpened} />
              </Hide>

              <a href="https://www.catho.com.br/">
                <LogoCatho src="https://static.catho.com.br/svg/site/logoCathoB2c.svg" alt="A gente trabalha pelo seu trabalho" />
              </a>

              <StyledHide tablet phone>
                <LinkList>
                  {
                    data.menu.desktop.links.map(link => (
                      <LinkListItem key={link.id}>
                        { !link.collapsedLinkGroup &&
                          <Link href={link.href}>
                            {link.text}
                          </Link>
                        }

                        { link.collapsedLinkGroup &&
                          link.text
                        }

                        { link.collapsedLinkGroup &&
                          <PopOverLinkList>
                            {
                              link.collapsedLinkGroup.map(collapsedlink => (
                                <PopOverLinkListItem key={collapsedlink.id}>
                                  { collapsedlink.text &&
                                    <PopOverLink href={collapsedlink.href}>
                                      { collapsedlink.text }
                                    </PopOverLink>
                                  }
                                  {
                                    collapsedlink.title &&
                                    <PopOverTitle>
                                      { collapsedlink.title }
                                    </PopOverTitle>
                                  }
                                </PopOverLinkListItem>
                              ))
                            }
                          </PopOverLinkList>
                       }

                      </LinkListItem>
                    ))
                  }
                </LinkList>
              </StyledHide>
            </MenuNavigation>
            <LoginBox handleLoginOpen={handleLoginOpen} loginOpened={loginOpened} />
          </StyledCol>
        </Row>
      </Container>
    </MenuDesktopContainer>
  </ThemeProvider>
);

MenuDesktop.propTypes = {
  data: PropTypes.instanceOf(Object),
  handleMenuOpen: PropTypes.func.isRequired,
  handleLoginOpen: PropTypes.func.isRequired,
  menuOpened: PropTypes.string,
  loginOpened: PropTypes.bool,
};

MenuDesktop.defaultProps = {
  data: null,
  menuOpened: null,
  loginOpened: false,
};

const MenuDesktopContainer = styled(Container)`
  border-bottom: 1px solid #f4f4f4;
  position: relative;
  padding: 0;

  &.opened {
    height: 50px;
  }
`;

const LogoCatho = styled.img`
  margin: ${props => props.theme.marginLogo};
  max-width: 85px;

  @media (max-width: ${BREAKPOINTS.desktop}px) {
    max-width: 55px;
  }
`;

const StyledHide = styled(Hide)`
  display: flex;
  height: 100%;
  width: 100%;
`;

const StyledCol = styled(Col)`
  background-color: ${props => props.theme.backgroundColor};
  display: flex;
  height: 70px;
  justify-content: space-between;

  @media (max-width: ${BREAKPOINTS.tablet}px) {
    height: 50px;
  }

  &.opened {
    animation: desktopMoveToHeaderBar 0.3s ease, desktopMoveToViewHeight 0.7s forwards ease 0.3s;
  }

  &.closed {
    animation: desktopMoveToViewHeightReverse 0.615s ease-out, desktopMoveToHeaderBarReverse 0.3s forwards ease-in 0.6s;
  }

  @keyframes desktopMoveToHeaderBar {
    0% { transform: translateY(0); }
    100% { transform: translateY(-35px); }
  }

  @keyframes desktopMoveToViewHeight {
    0% { transform: translateY(-35px); }
    100% { transform: translateY(calc(100vh - 50px - 35px)); }
  }

  @keyframes desktopMoveToHeaderBarReverse {
    0% { transform: translateY(-35px); }
    100% { transform: translateY(0); }
  }

  @keyframes desktopMoveToViewHeightReverse {
    0% { transform: translateY(calc(100vh - 50px - 35px)); }
    100% { transform: translateY(-35px); }
  }
`;

const MenuNavigation = styled.div`
  display: flex;
  align-items: center;
`;

const LinkList = styled.ul`
  display: flex;
  flex-direction: row;
  font-family: ${props => props.theme.fontFamily};
  font-size: ${props => props.theme.linkFontSize};
  height: 100%;
  list-style-type: none;
  margin: 0;
  padding: 0;
`;

const LinkListItem = styled.li`
  align-items: center;
  border-bottom: ${props => props.theme.menuHover.borderSize} solid transparent;
  color: ${props => props.theme.primaryColor};
  cursor: pointer;
  display: flex;
  margin-right: 20px;
  position: relative;

  &:hover {
    border-bottom: ${props => props.theme.menuHover.borderSize} solid ${props => props.theme.menuHover.borderColor};
    ul {
      visibility: visible;
      opacity: 1;
    }
  }

  &:last-child {
    &:hover {
      border-bottom:${props => props.theme.menuHover.borderSize} solid transparent;
    }
  }

  @media (max-width: ${BREAKPOINTS.desktop}px) {
    font-size:11px;
  }
`;

const Link = styled.a`
  color: ${props => props.theme.primaryColor};
  text-decoration: none;

  &.selected {
    cursor: default;
    color: ${props => props.theme.tertiaryColor};
    font-weight: bold;
  }
`;

const PopOverLinkList = styled.ul`
  background-color: #FFF;
  border-top:2px solid #005da4;
  box-shadow:rgba(0, 0, 0, 0.2) 0px 0px 10px 0px;
  display: flex;
  font-family: ${props => props.theme.fontFamily};
  font-size: ${props => props.theme.linkFontSize};
  flex-direction: column;
  left: calc(50% - calc(270px/2));
  list-style-type: none;
  margin: 0;
  opacity:0;
  padding:11px 0;
  position: absolute;
  transition:opacity 0.5s ease-out;
  top: 100%;
  visibility: hidden;
  width:285px;
  z-index: 9999;

  &:before {
    color: #005da4;
    content: "\\25b2";
    font-size: 22px;
    left: calc(50% - 12px);
    padding-left: 0;
    position: absolute;
    top: -17px;
    transform: scale(1.5, .7);
    z-index: 0;
  }
`;

const PopOverLinkListItem = styled.li`
  display: flex;
  padding: 4px 0;
`;

const PopOverTitle = styled.span`
  background-color: #f5f5f5;
  font-size: 10px;
  padding: 8px 15px 4px;
  text-transform: uppercase;
  width: 100%;
`;

const PopOverLink = styled.a`
  color: ${props => props.theme.primaryColor};
  padding: 8px 15px;
  text-decoration: none;
  width: 100%;
`;

export default MenuDesktop;
