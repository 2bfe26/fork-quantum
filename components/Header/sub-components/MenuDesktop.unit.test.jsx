import React from 'react';
import renderer from 'react-test-renderer';
import MenuDesktop from './MenuDesktop';
import data from '../data';

describe('MenuDesktop component', () => {
  it('Should match the snapshot', () => {
    const tree = renderer.create(<MenuDesktop
      data={data}
      handleLoginOpen={() => {}}
      handleMenuOpen={() => {}}
      menuOpened={null}
      loginOpened={false}
    />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
