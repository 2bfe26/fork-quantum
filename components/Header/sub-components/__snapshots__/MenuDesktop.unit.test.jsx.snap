// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuDesktop component Should match the snapshot 1`] = `
<div
  className="sc-VigVT fLlHKj sc-bdVaJa OkJiS"
>
  <div
    className="sc-bdVaJa cSQWIE"
  >
    <div
      className="sc-bwzfXH dKHdhw"
    >
      <div
        className="sc-jzJRlG jfmbzt sc-htpNat bLNipZ"
      >
        <div
          className="sc-cSHVUG hDNPdi"
        >
          <div
            className="sc-bxivhb jXYxts"
          >
            <div
              className="sc-ifAKCX bSVAvB"
              onClick={[Function]}
            >
              <span
                className="sc-EHOje iMdrmK"
              />
              <span
                className="sc-bZQynM ijIEHh"
              />
              <span
                className="sc-gzVnrw erHGXT"
              />
            </div>
          </div>
          <a
            href="https://www.catho.com.br/"
          >
            <img
              alt="A gente trabalha pelo seu trabalho"
              className="sc-jTzLTM iTDoST"
              src="https://static.catho.com.br/svg/site/logoCathoB2c.svg"
            />
          </a>
          <div
            className="sc-fjdhpX jiIyNK sc-bxivhb XNLln"
          >
            <ul
              className="sc-kAzzGY geykMb"
            >
              <li
                className="sc-chPdSV dzwkXp"
              >
                <a
                  className="sc-kgoBCf eteDkT"
                  href="/vagas/"
                >
                  Buscar vagas
                </a>
              </li>
              <li
                className="sc-chPdSV dzwkXp"
              >
                <a
                  className="sc-kgoBCf eteDkT"
                  href="/profissoes/"
                >
                  Guia de profissões
                </a>
              </li>
              <li
                className="sc-chPdSV dzwkXp"
              >
                <a
                  className="sc-kgoBCf eteDkT"
                  href="/por-dentro/"
                >
                  Por dentro das empresas
                </a>
              </li>
              <li
                className="sc-chPdSV dzwkXp"
              >
                Soluções para candidatos
                <ul
                  className="sc-kGXeez jimJLB"
                >
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="/aviso-vagas/listagem/"
                    >
                      Aviso de vagas
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="https://seguro.catho.com.br/inclusao/curriculo.php?nova_inclusao=1&usr[plano_id]=1&cur[perfil_id]=1&cur[use_personalizado]=0&a=0&ppd=0&tipoinclusao=807"
                    >
                      Envie seu Currículo por 7 dias grátis
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="/profissoes/"
                    >
                      Guia de profissões
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="/por-dentro/"
                    >
                      Por dentro das empresas
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="https://www.catho.com.br/portaldocurriculo/"
                    >
                      Portal do currículo
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="https://www.catho.com.br/estagio/"
                    >
                      Vagas de estágio
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <span
                      className="sc-dxgOiQ icpgaR"
                    >
                      Exclusivo para assinantes
                    </span>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="/cursos-rapidos/"
                    >
                      Cursos rápidos
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="https://cadastro.catho.com.br/"
                    >
                      Cadastro Catho
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="/consvirtual/"
                    >
                      Consultor virtual
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="/cadastrar-curriculo/"
                    >
                      Vantagens de assinar a Catho
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <span
                      className="sc-dxgOiQ icpgaR"
                    >
                      Apoio ao candidato
                    </span>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6503&p=12&url1=1&url=servicoscatho/carreira/elaboracao-de-curriculo/"
                    >
                      Elaboração de currículo
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6507&p=4&url1=1&url=servicoscatho/carreira/analise-de-curriculo/"
                    >
                      Análise de currículo
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6509&p=16&url1=1&url=servicoscatho/carreira/simulacao-de-entrevista/"
                    >
                      Simulação de entrevista
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="https://parceiros.catho.com.br/gop.php?origem=cat&a=6505&p=13&url1=1&url=servicoscatho/carreira/carta-de-apresentacao/"
                    >
                      Elaboração de carta de apresentação
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="/curso/"
                    >
                      Cursos
                    </a>
                  </li>
                  <li
                    className="sc-kpOJdX hCRoSg"
                  >
                    <a
                      className="sc-ckVGcZ fbQkxY"
                      href="/servicoscandidatos/"
                    >
                      Todas as soluções
                    </a>
                  </li>
                </ul>
              </li>
            </ul>
          </div>
        </div>
        <ul
          className="sc-htoDjs kkROuE"
        >
          <li
            className="sc-dnqmqq igIpdT"
          >
            <a
              className="sc-iwsKbI ggKTXP"
              href="https://seguro.catho.com.br/inclusao/curriculo.php?nova_inclusao=1&usr%5Bplano_id%5D=1&cur%5Bperfil_id%5D=1&cur%5Buse_personalizado%5D=0&a=0&ppd=0&tipoinclusao=903"
            >
              Assine
            </a>
          </li>
          <li
            className="sc-dnqmqq igIpdT"
          >
            <button
              className="sc-gZMcBi dvYuZS"
              onClick={[Function]}
            >
              Login
            </button>
            <div
              className="sc-gqjmRU hAFoso"
            >
              <iframe
                frameBorder="0"
                scrolling="no"
                src="https://safe.catho.com.br/login/tipo/tooltip/"
                title="LoginBox"
              />
            </div>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
